OnBaseDocsApi/Controllers/OnBaseDocsController.cs

```diff
@@ -108,22 +294,31 @@ IHttpActionResult DocumentResult(Document doc)
 
             return Ok(new DataResult<DocumentAttributes>
             {
-                Data = new DocumentAttributes
+                Data = new DataResource<DocumentAttributes>
                 {
-                    ID = doc.ID,
-                    CreatedBy = doc.CreatedBy.ID,
-                    DateStored = doc.DateStored,
-                    DocumentDate = doc.DocumentDate,
-                    Status = doc.Status.ToString(),
-                    Name = doc.Name,
-                    DocumentType = doc.DocumentType.Name,
-                    DefaultFileType = doc.DefaultFileType.Name,
-                    LatestAllowedRevisionID = doc.LatestAllowedRevisionID,
-                    Keywords = keywords,
+                    Id = doc.ID,
```

Comment:
This should be a string https://jsonapi.org/format/#document-resource-object-identification

Reply:
Done.